### PR TITLE
S2U-35 Conversion: Lessons - View items based on grade

### PR DESCRIPTION
**Pull request by stetsche**
Conversion scripts for PR: https://github.com/sakaiproject/sakai/pull/11856/

Jira: https://sakaiproject.atlassian.net/browse/S2U-35

### CODE REVIEW

**Comment by stetsche** [comment]
I tested the MySQL script.
@jesusmmp could you check if the oracle script looks allright, please?

File: docs/conversion/s2u_features/S2U-35-oracle.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  ARGUMENT varchar2(999) DEFAULT NULL,
+  SITE_ID varchar2(36) NOT NULL,
+  TOOL_ID varchar2(99) NOT NULL,
+  ITEM_ID varchar2(99) DEFAULT NULL,
+  PRIMARY KEY (ID),
+);
+CREATE INDEX IDX_CONDITION_SITE_ID ON COND_CONDITION (SITE_ID);
+
+CREATE TABLE COND_PARENT_CHILD (
+  PARENT_ID varchar(36) NOT NULL,
```
Comment (jesusmmp) [inline docs/conversion/s2u_features/S2U-35-oracle.sql:15]: varchar2

File: docs/conversion/s2u_features/S2U-35-oracle.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  CHILD_ID varchar(36) NOT NULL,
```
Comment (jesusmmp) [inline docs/conversion/s2u_features/S2U-35-oracle.sql:16]: varchar2

File: docs/conversion/s2u_features/S2U-35-oracle.sql (hunk 1 above)
Comment (stetsche) [reply docs/conversion/s2u_features/S2U-35-oracle.sql]: fixed, thanks!

File: docs/conversion/s2u_features/S2U-35-oracle.sql [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+CREATE INDEX IDX_CONDITION_SITE_ID ON COND_CONDITION (SITE_ID);
+
+CREATE TABLE COND_PARENT_CHILD (
+  PARENT_ID varchar2(36) NOT NULL,
+  CHILD_ID varchar2(36) NOT NULL,
+  PRIMARY KEY (PARENT_ID, CHILD_ID),
+  CONSTRAINT FK_CHILD_ID_CONDITION_ID FOREIGN KEY (CHILD_ID) REFERENCES COND_CONDITION (ID),
+  CONSTRAINT FK_PARENT_ID_CONDITION_ID FOREIGN KEY (PARENT_ID) REFERENCES COND_CONDITION (ID)
+);
+CREATE INDEX FK_CHILD_ID_CONDITION_ID (CHILD_ID),
```
Comment (jesusmmp) [inline docs/conversion/s2u_features/S2U-35-oracle.sql:21]: CREATE INDEX FK_CHILD_ID_CONDITION_ID ON COND_PARENT_CHILD (CHILD_ID);

Comment (stetsche) [reply docs/conversion/s2u_features/S2U-35-oracle.sql]: Thanks!

**Review by jesusmmp — APPROVED** [review]

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Thanks for the review @jesusmmp !

**Pull request merged**